code/src/components/Layout.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+const Layout = ({ children }) => {
+  return (
+    <div className="layout">
+      <header className="layout-header">Borzoi survey</header>
+      <main className="layout-content">{children}</main>
+      <footer className="layout-footer"><a href="https://www.amandas.dev/" className="footer-link">by amandas.dev 2023</a></footer>
```

Comment:
The link doesn't work, maybe it's for the future.

Reply:
Yeah, I thought I'd have the domain set up before finishing this project but I still haven't gotten around to it 😅